src/sample/Main.java

```diff
@@ -16,40 +17,42 @@
 public class Main extends Application {
 
     public static void main(String[] args) {
+        boardGenerator i = new boardGenerator(18);
+
         String inputFile = "examples/example.in";
         String outputFile = null;
 
-        if (args.length == 2){
+        if (args.length == 2) {
             inputFile = args[0];
             outputFile = args[1];
         }
 
         DataInput di = new DataInput();
 
-
-        di.ReadBoard(inputFile);
+        di.readString(i.toString());
```

Comment:
I think we shouldn't commit changes in Main.java that are used only for help in local debug etc.